client/planning-extension/src/extension.ts

```diff
@@ -135,6 +137,28 @@ const extension: IExtension = {
                                     ) {
                                         const event = new CustomEvent('planning:unlinkfromcoverage', {detail: {item}});
 
+                                        window.dispatchEvent(event);
+                                    }
+                                },
+                            },
+                            {
```

Comment:
I'm wondering whether this would appear in authoring-angular as well?

Reply:
Extensions API have access to `authoringReactViewEnabled` variable. What about using it to conditionally push the action?

Alternativelly, what about if we dropped the old action and kept only the new one? I did that with [planning details widget](https://github.com/superdesk/superdesk-planning/pull/2078) - dropped the angular version of a widget - and the react one works in both - angular and react based authoring.